scraper/src/main/java/com/example/cosc345/scraper/scrapers/woocommerce/KingsFishMarketScraper.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.cosc345.scraper.scrapers.woocommerce
+
+import com.example.cosc345.scraper.scrapers.generic.WooCommerceScraper
+import com.example.cosc345.shared.models.Region
+import com.example.cosc345.shared.models.Retailer
+import com.example.cosc345.shared.models.Store
+
+/**
+ * The Kings Fish Market specific implementation of the [WooCommerceScraper], which essentially just passes information through to it.
+ */
```

Comment:
@constructor Create a new instance of this scraper.

Reply:
Was something I did initially, but really it wasn't needed and just bloats up the comments

Reply:
Fair